### ISSUE TITLE
A better approach for PRs from forks (workflow commands via stdout)

### ISSUE THREAD

**Issue by jnm2** [bradennapier/eslint-plus-action#59]
If you write workflow commands to stdout instead of making web API calls which depend on the token, the authentication issue mentioned at https://github.com/bradennapier/eslint-plus-action#handle-forked-prs goes away and the performance may even be better.

Example from docs page (https://docs.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-a-warning-message):

```
echo "::warning file=app.js,line=1,col=5::Missing semicolon"
```

Both warning and error inline annotations can be created this way. A GitHub action using this approach: https://github.com/cschleiden/jest-github-actions-reporter

![](https://raw.githubusercontent.com/cschleiden/jest-github-actions-reporter/master/img/annotations.png)

**Comment by bradennapier** [comment]
Yes that is new -- this was made back when they didn't have support for actions in forked repos so a lot of the craziness is built around a method i found to make it work properly.
